types/three/three-core.d.ts

```diff
@@ -1447,6 +1447,15 @@ export class Geometry extends EventDispatcher {
     dispatchEvent(event: { type: string; [attachment: string]: any; }): void;
 }
 
+/**
+ * @see <a href="https://github.com/mrdoob/three.js/blob/master/examples/js/BufferGeometryUtils.js">examples/js/BufferGeometryUtils.js</a>
```

Comment:
I don't think it's necessary to wrap the link in an anchor tag. See [\@see documentation](http://usejsdoc.org/tags-see.html)

Reply:
Much of the rest of the three-core.d.ts classes seemed to follow the see convention (that's why I went with that at first). Any reason in particular we want to deviate in this case?

Reply:
I could be wrong, but I suspect the other instances are incorrectly formatted. This is a minor nitpick. I'll let the maintainers decide.

Reply:
Sounds good. Happy to change it to whatever format is best if requested.